source/guide_keycloak.rst

```diff
@@ -143,144 +160,38 @@ Setup Backend
 
-Tuning
-======
-
-For further information on configuration and usage read the `Keycloak Documentation`_.
-
-Updates
-=======
-
-.. note:: Check the update feed_ regularly to stay informed about the newest version.
-
-To update Keycloak you need to remove the symlink to :file:`~/keyclok-current` first.
-
-::
-
- [isabell@stardust ~]$ rm ~/keycloak-current
- [isabell@stardust ~]$
-
-Now install the new version as described under :lab_anchor:`Installation <guide_keycloak.html#installation>`, but without the configuration steps.
-
-Migrate data
-------------
-
-Copy the :file:`standalone` directory from the previous installation (version 12.0.0 in this example) over the directory in the new installation:
 
-::
-
- [isabell@stardust ~]$ cp -r keycloak-12.0.0/standalone keycloak-current
- keycloak: stopped
- [isabell@stardust ~]$
-
-You also need to copy any custom modules that have been added to the :file:`modules` directory in the same way.
-
-For some versions manual migration steps are needed: `Migration Guide`_.
-
-Stop the process of the previous version
-----------------------------------------
-
-
-First stop the daemon:
-
-::
-
- [isabell@stardust ~]$ supervisorctl stop keycloak
-  keycloak: stopped
- [isabell@stardust ~]$
-
-Unfortunately stopping the daemon is not enough so you need to identify the running Keycloak process. The command starts with ``java -D[Standalone]``.
+A note regarding identity providers
+===================================
 
+In case you are intending to use Microsoft as an identity provider with keycloak, they will require you to verify your domain. An easy way to do that is to go through this process:
+* temporarily delete the web backend using ``userspace web backend del isabell.uber.space``
+* create a directory ``.well-known`` inside your ``/var/www/virtual/isabell/html``
+* create a file ``microsoft-identity-association.json`` in that ``.well-known`` directory with the following contents:
 .. code-block::
- :emphasize-lines: 4
-
- [isabell@stardust ~]$ ps aux
- USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
- isabell   1138  0.0  0.0 219848 17536 ?        Ss   Nov02  11:40 /opt/uberspace/python-venv/bin/python /usr/bin/supervisord -c /etc/supervisord.conf
- isabell   1786  0.4  1.5 1659844 477116 ?      Sl   Nov02 116:23 java -D[Standalone] -server -Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -Djava.net.preferIPv4Stack=true - Djboss.modules.system.pkgs=org.jboss.byteman
- isabell  23108  0.0  0.0 116912  3556 pts/6    Ss   22:25   0:00 -bash
- isabell  26803  0.0  0.0 155452  1844 pts/6    R+   23:18   0:00 ps aux
- isabell  32155  0.0  0.0 737800 27388 ?        Ss   Nov18   0:11 php-fpm: master process (/opt/uberspace/etc/isabell/php-fpm.conf)
- [isabell@stardust ~]$
-
-And kill the process. Replace ``PID`` by the PID identified in the previous step.
-
-::
-
- [isabell@stardust ~]$ kill -SIGKILL PID
- [isabell@stardust ~]$
-
-Run the upgrade script
```

Comment:
Could you please provide some feedback to these questions? Otherwise, we'll close this PR next month.

Reply:
I had removed everything that, in its former form, does not apply to the newest Keycloak version any more. We need to completely rewrite the update guide. So far, my intention was to ensure the installation steps are properly reflected as they had to change quite dramatically, and that everything that would break any new installation is removed.